stats/system_tag.go

```diff
@@ -70,6 +71,17 @@ func (i *SystemTagSet) Map() TagSet {
 	return m
 }
 
+// SetString is returns comma separeted list of the string representation of all values in the set
```

Comment:
Typo in "is returns".

And also, should this method named `List()`, for alignment with `Map()` above/

Reply:
Fixed the typo and did some .. performance optimizations (not actually needed I am sure). 

I don't think it's worth it to add a generic way to get a list of the set values from the SystemTags ... this is literally the only place where it's used ...